gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -0,0 +1,122 @@
+require 'postgres_ha_admin/failover_databases'
+require 'postgres_ha_admin/database_yml'
+require 'pg'
+require 'linux_admin'
+
+module PostgresHaAdmin
+  class FailoverMonitor
+    FAILOVER_ATTEMPTS = 60
+    DB_CONNECTED_CHECK_FREQUENCY = 5.minutes
+    FAILOVER_CHECK_FREQUENCY = 1.minute
```

Comment:
Can we make these actually configurable?

Maybe put them in a yml config file that we can specify in the `initialize` method?